app/components/inputgroup/inputgroup.js

```diff
@@ -17,15 +17,19 @@
 var React = require('react');
 var _ = require('lodash');
 
+var DatePicker = require('../datepicker');
+
 // Input with label and validation error message
 var InputGroup = React.createClass({
   propTypes: {
     name: React.PropTypes.string,
```

Comment:
Should any/all of these props be `isRequired`? The original component may date from before we decided on enforcing `propTypes` as part of our style.